index.html

```diff
@@ -189,6 +189,51 @@ <h4 class="title">Skills</h4>
         </div>
     </section>
     </div>
+    <!-- contact form -->
+    <footer>
+        <section class="form-container" id="form">
+            
+            <h2 class="form-text">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <div class="form">
+                <form
+                action="https://formspree.io/f/mqkngnbr"
+                method="POST"
+              >
+                    <input type="text" class="text-form" id="name" name="username" required 
+                    maxlength="30" placeholder=" Fullname"><br>
+              
+                    <input type="Email" class="mail" id="email" name="email" required
+                     placeholder=" EmailAddress"><br>
+                
+                    <textarea class="subject" name="subject" required maxlength="500" placeholder=" Enter text here.." ></textarea>
+               
+                <button class="form-button" type="submit">Get In Touch</button>
```

Comment:
### Your Button
![student-form](https://user-images.githubusercontent.com/93039359/186212212-4bb175cc-41ad-42b0-8d1b-912a5f17180b.png)

### Figma Button
![figma](https://user-images.githubusercontent.com/93039359/186212258-5a4f64e1-014f-4b2b-8e0b-88edf27e251d.png)


- [ ] Your button is not in the center as shown in the Figma template. Please make your button in center to follow the Figma template :+1: